fedmsg_meta_fedora_infrastructure/tests/github.py

```diff
@@ -661,143 +664,1198 @@ class TestGithubIssueComment(Base):
                 "default_branch": "develop",
                 "html_url": "https://github.com/fedora-infra/github2fedmsg",
                 "has_issues": True,
-                "master_branch": "develop",
-                "forks": 0,
-                "open_issues_count": 1,
-                "watchers": 2,
-                "name": "github2fedmsg",
+                "master_branch": "develop",
+                "forks": 0,
+                "open_issues_count": 1,
```

Comment:
Is this changing an existing test? If so, I don't think we should, add new one, deprecate the old one is fine, but we shouldn't change existing ones, imho.